build/tasks/validate.js

```diff
@@ -307,5 +301,170 @@ function validateRule({ tags, metadata }) {
   if (help.toLowerCase().includes(prohibitedWord)) {
     issues.push(`metadata.help can not contain the word '${prohibitedWord}'.`);
   }
+
+  issues.push(...findTagIssues(tags));
   return issues;
 }
+
+const uniqueTags = ['ACT', 'experimental', 'review-item', 'deprecated'];
+
+const categories = [
+  'aria',
+  'color',
+  'forms',
+  'keyboard',
+  'language',
+  'name-role-value',
+  'parsing',
+  'semantics',
+  'sensory-and-visual-cues',
+  'structure',
+  'tables',
+  'text-alternatives',
+  'time-and-media'
+];
+
+const standardsTags = [
+  {
```

Comment:
It's non-obvious that `findTagIssues` relies on the `WCAG` entry being first in the list, maybe leave a comment noting that that's important?